packages/openapi-typescript/tsconfig.examples.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["examples"],
+  "exclude": ["examples/digital-ocean-api.ts"]
```

Comment:
An overdue addition to v7 is enforcing that the generated snapshots are valid TypeScript. Before this more-or-less happened manually, but now it’s enforced in CI.

However, the DigitalOcean schema is a bit of a love–hate relationship. On the one hand, it does really stress-test many of the features that make this library robust and flexible. But on the other, the schema itself has errors 🙃.

Unfortunately, this means we can’t use DigitalOcean’s schema to check errors until they themselves resolve the problems with their schema. But I still like having it around for snapshot testing to just make sure this library _can_ generate _something_ there without blowing up.